### ISSUE TITLE
CSS support for Malvolio

### ISSUE THREAD

**Issue by teymour-aldridge** [lovelace-edu/lovelace#58]
Add support for CSS to Malvolio.

**Comment by teymour-aldridge** [comment]
So these are my initial thoughts for the API:

```rust
H1::new().map(BlueText::apply).to_html();

#[derive(CSS)]
#[color="blue"]
struct BlueText;

trait Apply<T> {
  fn apply(&self, item: &mut T) -> &mut T;
}
```

```rust
/*Autogenerated code might look like this*/
impl Apply<H1> for BlueText {
  fn apply(&self, h1: &mut H1) -> &mut H1 {
    h1.attribute(class: Class::from("autogenerated-class-name"))
  }
}
impl Apply<Head> for BlueText {
  fn apply(&self, h1: &mut Head) -> &mut H1 {
    head.child(Style::new(".autogenerated-class-name {color: \"blue\"}"))
  }
}
```

**Comment by teymour-aldridge** [comment]
This was fixed in #69
